broken-links-script/package.json

```
@@ -0,0 +1,19 @@
+{
+  "name": "broken-links-script",
+  "version": "1.0.0",
+  "type": "module",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "test:headless": "cypress run --browser chrome --spec 'cypress/e2e/link-checker.cy.js'"
```

Why not use the main `test` script to run your tests, and maybe also include the prerequisite command (we want the tests to always run with the latest state of links, right?)?
```suggestion
    "test": "node Extractlinks.js && cypress run --browser chrome --spec 'cypress/e2e/link-checker.cy.js'"
```